src/robot_gnss/gnss_math.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+from math import acos, asin, sin, sqrt, degrees, radians
+from numpy import nan
+
+# https://en.wikipedia.org/wiki/Great-circle_distance
```

Comment:
You should not refer to Wikipedia, this is not the most scientific community :)

Reply:
Fixed